hack/tools.go

```diff
@@ -34,9 +34,6 @@ import (
 	// Migration job.
 	_ "knative.dev/pkg/apiextensions/storageversion/cmd/migrate"
 
-	// Mako stub
-	_ "knative.dev/pkg/test/mako/stub-sidecar"
```

Comment:
I guess we need to clean up https://github.com/knative/hack/blob/main/performance-tests.sh#L26 and other repos?

Reply:
I think so true. I don't know, but as google stopped working on that, not sure if we even should continue to use it in Knative at all.

Reply:
Added todos above, well do those in seperate PRs.